configs.py

```diff
@@ -54,6 +52,7 @@ class BaseConfig(BaseModel):
     num_classes: int = 5
     random_seed: int = 100
     save_checkpoints: bool = False
+    num_workers: int = 8
```

Comment:
I think it's still best to have a different number of workers, eventually, if we parallelize finetuning we are going to have a lot of workers for the general training and fewer working for offline/online finetuning